raddb/policy.d/accounting

```diff
@@ -64,23 +62,17 @@ acct_unique {
 	#  is not included
 	#
 	else {
-		update request {
-			&Acct-Unique-Session-Id := "%{hex:%{md5:%{string:%{User-Name},%{Acct-Multi-Session-ID},%{Acct-Session-ID},%{%{NAS-IPv6-Address}:-%{NAS-IP-Address}},%{NAS-Identifier},%{NAS-Port-ID},%{NAS-Port}}}}"
-		 }
+		&request.Acct-Unique-Session-Id := %{hex:%{md5:%{string:%{User-Name},%{Acct-Multi-Session-ID},%{Acct-Session-ID},%{%{NAS-IPv6-Address}:-%{NAS-IP-Address}},%{NAS-Identifier},%{NAS-Port-ID},%{NAS-Port}}}}
 	}
 
-	update request {
-		&Tmp-String-9 !* ANY
-	}
+	&request.Tmp-String-9 !* ANY
```

Comment:
This should also be `&request -= &Tmp-String-9[*]`

You can take a pass looking for `!*` and fix them.